File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        boolean b = false;
        Set<Integer> set = new HashSet<>();
        for(int i = 0;i<nums.length;i++){
            if(!set.add(nums[i]))
                b = true;
        }
        return b;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        freq = {}
        for i in nums:
            key = i
            if key not in freq:
                freq[key] = 0
            freq[key] += 1

        n = []
        top = sorted(freq, key=freq.get, reverse=True)
        for i in range(k):
            n.append(top[i])
        return n